src/pykeen/models/unimodal/trans_h.py

```diff
@@ -87,27 +88,18 @@ def __init__(
             random_seed=random_seed,
             regularizer=regularizer,
         )
-
         self.scoring_fct_norm = scoring_fct_norm
 
         # embeddings
-        self.normal_vector_embeddings = Embedding.init_with_device(
+        self.normal_vector_embeddings = Embedding.from_specification(
             num_embeddings=triples_factory.num_relations,
             embedding_dim=embedding_dim,
-            device=self.device,
```

Comment:
why not pass device?

Reply:
I think it might be worth to get rid of passing around device everywhere, and move the model once to the device it is needed (e.g. at the start of the training loop).

Reply:
It increases fault tolerance, since pytorch will take care for us of moving every submodule / parameter / buffer to the device.

Reply:
Agreed